solutions/swb-reference/src/services/dataSetService.test.ts

```diff
@@ -62,10 +62,10 @@ describe('DataSetService', () => {
       id: '12345678-1234-1234-1234-123456789012',
       roles: []
     };
-    projectId = 'proj-projectId';
+    projectId = 'proj-12345678-1234-1234-1234-123456789012';
```

Comment:
i'd suggest you save `const mockId = 12345678-1234-1234-1234-123456789012` at top most describe along with mockProjId, mockDatasetId

and call these variables in each test suite instead of creating separate ones in each test

Reply:
Agreed, I decided to roll back the changes here so as to not add more confusion to the IDs.